### PR TITLE
fix(stepper): adds styles for windows hcm

### PR DESCRIPTION
**Pull request by oliverschuerch**

### CODE REVIEW

**Review by gfellerph — CHANGES_REQUESTED** [review]
@oliverschuerch can you decrease the line thickness to 2px to be more consistent with other borders?

**Comment by oliverschuerch** [comment]
@gfellerph I fixed the indicator-size in forced-color/high-contrast mode, to match the size in normal mode.
This made some of your suggestions obsolete. Please have a look.

File: .changeset/hungry-news-swim.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/web-styles': patch
+---
+
+adds styles for component stepper in forced-color/high-contrast mode
```
Comment (gfellerph) [inline .changeset/hungry-news-swim.md:5]:
```suggestion
Adds styles for component stepper in forced-color/high-contrast mode
```

**Review by gfellerph — APPROVED** [review]
Looks good now!

**Pull request merged**